tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/security/SecurityTest.java

```diff
@@ -2564,4 +2582,21 @@ private void checkUserSendNoReceive(final String queue, final ClientSession conn
          fail("Invalid Exception type:" + e.getType());
       }
    }
+
+   public static class MD5SensitiveDataCodec implements SensitiveDataCodec<String> {
```

Comment:
Same general comment as before. The DigestUtils class being used in the impl has many options, so I see no reason to use MD5 at this stage, even if it is only for a test impl.